.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "ext/cli"]
+	path = ext/cli
+	url = ../cli.git
+	branch = new_docs_site
```

Comment:
Using this comment to keep the PR open until coordination between the `cli` repo and the corresponding changes there (https://github.com/phylum-dev/cli/pull/1327) are complete.

Reply:
Discussions with @kylewillmon and @furi0us333 today resulted in a green light to proceed with this cutover. The changes here are good to be merged now.